react_juce/core/View.cpp

```diff
@@ -191,6 +191,10 @@ namespace reactjuce
             auto c = juce::Colour::fromString(colorString);
             float borderWidth = props[borderWidthProp];
 
+            auto myAreaRect = getLocalBounds();
```

Comment:
@nick-thompson , This is a mistake left over from debugging a thing. Will remove.